docs/source/python/json.rst

```diff
@@ -38,6 +38,9 @@ The features currently offered are the following:
   such as ``my_data.json.gz``)
 * sophisticated type inference (see below)
 
+.. note::
+   Currently only the line-delimited JSON (LDJSON) format is supported.
```

Comment:
I am not sure how common the "LDJSON" abbreviation is. If you google that, I rather get hits for JSON-LD, which is json for linked data. 
Searching for line delimited json gives "newline-delimited JSON (NDJSON)" (http://ndjson.org/) and "JSON Lines" (http://jsonlines.org/). Wiki mentions all three terms, though: https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON